coverage.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+printf -- 'running flake8 \n';
+flake8 questionpy
+flake8 tests
+
+printf -- 'running pylint \n';
+pylint questionpy
+pylint tests
+
+printf -- 'running pytest \n';
+pytest tests
+
+printf -- 'running mypy \n';
+mypy questionpy
+mypy tests
```

Comment:
1. Die Semikolons sind nicht nötig
2. Das ganze läuft etwas schneller, wenn wir `flake8 questionpy tests` etc, also jeweils beides im selben Befehl, machen
3. Wollen wir, dass das Script weiter läuft, wenn es z.B. in flake8 einen Fehler gab? Sonst könnte man das mit `set -o errexit` ganz oben unterbinden. Ist aber Abwägungssache